recipes/gcc/gcc-5.3/aarch64-multilib-dirnames.patch

```diff
@@ -0,0 +1,12 @@
+--- gcc-5.3.0/gcc/config/aarch64/t-aarch64-linux.bak	2016-01-07 13:52:31.113702383 +0000
++++ gcc-5.3.0/gcc/config/aarch64/t-aarch64-linux	2016-01-07 13:55:39.470923680 +0000
+@@ -22,7 +22,7 @@
+ LIB1ASMFUNCS = _aarch64_sync_cache_range
+ 
+ AARCH_BE = $(if $(findstring TARGET_BIG_ENDIAN_DEFAULT=1, $(tm_defines)),_be)
+-MULTILIB_OSDIRNAMES = mabi.lp64=../lib64$(call if_multiarch,:aarch64$(AARCH_BE)-linux-gnu)
++MULTILIB_OSDIRNAMES = mabi.lp64=.$(call if_multiarch,:aarch64$(AARCH_BE)-linux-gnu)
```

Comment:
I honestly can remember why it ended up like this :-) But I can see your point so I will do some testing.